core-field-editors/src/_shared/boot.js

```diff
@@ -0,0 +1,10 @@
+import {init} from 'contentful-ui-extensions-sdk'
+
+export default function makeBooter (create) {
+  return function boot () {
+    init((api) => {
+      api.window.startAutoResizer()
```

Comment:
Doing this by default doesn't feel super right to me since auto-sizing doesn't work well with dropdowns or absolutely positioned pop-ins. Explicit calls make that somehow clearer IMO.